js/core/contrib.js

```diff
@@ -0,0 +1,98 @@
+// Module core/contrib
+// Fetches names of contributors from github and uses them to fill
+// in the content of elements with key identifiers:
+// #gh-commenters: people having contributed comments to issues.
+// #gh-contributors: people whose PR have been merged.
+// Spec editors get filtered out automatically.
+
+define(
+    ["github"],
+    function (github) {
+        return {
+            run: function (conf, doc, cb, msg) {
+                function theEnd () {
+                    msg.pub("end", "core/contrib");
+                    cb();
+                }
+                
+                function prop(prop) {
+                    return function (o) {
+                        return o[prop];
+                    };
+                }
+                
+                function slice(args) {
+                    return Array.prototype.slice.call(args, 0)
+                }
+
+                function findUsers() {
+                    var users = {};
+                    slice(arguments).forEach(function (things) {
```

Comment:
Nit: this nested .forEach bothers me, tbh. These kinds of functions are supposed to be chained, not nested. It probably means that the first forEach is not what you want here?